README.md

```diff
@@ -46,13 +55,45 @@ JupyterHubSSH is made up of two main components:
 	$ python -m jupyterhub_ssh
 	```
 
-## Instructions on how to use it
+### Kuberbetes based deployment
+If your JupyterHub was deployed using Kubernetes, you can use the Helm charts available in this repo to deploy JupyterHub SSH & SFTP
+directly into your Kubernetes cluster.
+
+	```bash
+	# Let helm the command line tool know about a Helm chart repository
+	# that we decide to name jupyterhub.
+	$ helm repo add jupyterhub https://yuvipanda.github.io/jupyterhub-ssh/
+	$ helm repo update
+
+	# Simplified example on how to install a Helm chart from a Helm chart repository
+	# named jupyterhub-ssh. See the Helm chart's documentation for additional details
+	# required.
+	$ helm install jupyterhub-ssh/<helm chart name> --version <helm chart version>
+	```
+
+## How to use it
 
+### How to SSH
 1. Login into your JupyterHub and go to `https://<hub-address>/hub/token`.
 2. Copy the token from JupyterHub.
 3. SSH into JupyterHub:
 	```bash
 	$ ssh <username-you-used>@<hub-address>
 	```
 4. Enter the token received from JupyterHub as a password.
-5. TADA :tada:
+5. TADA :tada: Now you have an interactive terminal! You can do anything you would generally interactively do via ssh: run editors,
+fully interactive programs, use the commandline, etc. Some features like non-interactive command running, tunneling, etc are currently
+unavailable.
+
+### How to SFTP
+1. Login into your JupyterHub and go to `https://<hub-address>/hub/token`.
+2. Copy the token from JupyterHub.
+3. Transfer file into Jupyterhub:
+	* Using the `sftp` command:
+		```bash
+		$ sftp -oPort=2222 <hub-username>@<hub-address>
```

Comment:
I think -o flag used to declare a ssh_option, and the option Port=2222, can be omitted, as the SSH entry to the SFTP server is exposed publicly on port 22, even though the container expose it at port 2222. But, I guess this depends a bit.

It will always depend on how the SFTP server is exposed, but I suggest to assume port 22 like would be correct if SFTP is deployed as part of the Helm chart.

```suggestion
		$ sftp <hub-username>@<hub-address>
```